### PR TITLE
Add script to categorize payload

### PR DESCRIPTION
**Pull request by xueqzhan**

### CODE REVIEW

File: scripts/rejected-payloads.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Here is a single update example:
+
+```
+% python ./rejected-payloads.py -d "your-dsn" categorize -t 4.11.0-0.nightly-2022-06-28-111405
+Please choose the reject reason for tag 4.11.0-0.nightly-2022-06-28-111405 from the following list:
+         1:           TEST_FLAKE
+         2:          CLOUD_INFRA
+         3:             RH_INFRA
+         4:   PRODUCT_REGRESSION
+         5:      TEST_REGRESSION
```
Comment (stbenjam) [inline scripts/rejected-payloads.md:63]: What are the differences between these two options? Could we maybe have a text explanation for each option?

File: sippy-ng/src/releases/ReleasePayloadTable.js [hunk 2]
```diff
@@ -116,6 +116,12 @@ function ReleasePayloadTable(props) {
         }
       },
     },
+    {
+      field: 'reject_reason',
+      headerName: 'RejectReason',
```
Comment (stbenjam) [inline sippy-ng/src/releases/ReleasePayloadTable.js:121]:
```suggestion
      headerName: 'Reject reason',
```

File: scripts/rejected-payloads.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+    categorize_parser = subparsers.add_parser('categorize', help='categorize a rejected payload')
+    categorize_parser.set_defaults(action='categorize')
+    categorize_parser.add_argument('-t', '--release_tag', help='Specifies a release payload tag, like 4.11.0-0.nightly-2022-06-25-081133', default=None)
+    categorize_parser.add_argument('-r', '--release', help='Specifies a release, like 4.11', default=None)
+    categorize_parser.add_argument('-s', '--stream', help='Specifies a stream, like nightly or ci', default=None)
+
+    args = vars(parser.parse_args())
+
+    #dsn = "postgresql://postgres:@localhost:5432/postgres"
```
Comment (stbenjam) [inline scripts/rejected-payloads.py:92]: Can you just make this the default option for --dsn?

File: scripts/rejected-payloads.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```
Comment (stbenjam) [inline scripts/rejected-payloads.py:1]: Could you ensure this is `chmod +x`

File: scripts/rejected-payloads.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+* "RH_INFRA",              # Payload was rejected because of other infrastructure failures
+* "PRODUCT_REGRESSION",    # Payload was rejected because of real product regression
+* "TEST_REGRESSION"        # Payload was rejected because of a test regression (typically from origin repo)
+```
+
+## Python Dependencies
+
+Use pip to install the following dependencies:
+
+`pip install argparse sqlalchemy`
```
Comment (stbenjam) [inline scripts/rejected-payloads.md:22]: WDYT of `sudo dnf install python3-sqlalchemy+postgresql` on Fedora? Personally I prefer not to pip if there's OS packages available.

**Review by stbenjam — COMMENTED** [review]
Nice, works well for me.  A couple of usability ideas.

1. Could `list` show already categorized releases too?
2. Could `categorize` also show the already categorized releases instead of removing them? I'd like to see what I've already categorized releases as, and be able to correct it if I make a mistake.

Specifically, I'd like the workflow to be like the below.  What do you think?

```
index     release tag                                       phase               reject reason       
1         4.11.0-0.nightly-2022-07-05-161230                Rejected                                
2         4.11.0-0.nightly-2022-07-04-160557                Rejected                                
3         4.11.0-0.nightly-2022-07-01-065600                Rejected                                
4         4.11.0-0.ci-2022-07-06-033033                     Rejected                                
5         4.11.0-0.ci-2022-07-05-192616                     Rejected                                
6         4.11.0-0.ci-2022-07-05-151913                     Rejected                                
Select tag between 1 and 6 to categorize, enter q to exit: 1
Please choose the reject reason for tag 4.11.0-0.nightly-2022-07-05-161230 from the following list:
         1:           TEST_FLAKE
         2:          CLOUD_INFRA
         3:             RH_INFRA
         4:   PRODUCT_REGRESSION
         5:      TEST_REGRESSION
Enter your selection between 1 and 5: 2
index     release tag                                       phase               reject reason       
1         4.11.0-0.nightly-2022-07-05-161230                Rejected            CLOUD_INFRA     
2         4.11.0-0.nightly-2022-07-04-160557                Rejected                                
3         4.11.0-0.nightly-2022-07-01-065600                Rejected                                
4         4.11.0-0.ci-2022-07-06-033033                     Rejected                                
5         4.11.0-0.ci-2022-07-05-192616                     Rejected                                
6         4.11.0-0.ci-2022-07-05-151913                     Rejected
Select tag between 1 and 6 to categorize, enter q to exit:                                 
```

**Comment by stbenjam** [comment]
> > 1. Could `list` show already categorized releases too?
> 
> list shows uncategorized releases by default. But with -a, all rejected payloads are showed. Let me know if you want the default to be showing all rejected (categorized and uncategorized) payload. I just think it might be too much.

Ah, that works.

> > 2. Could `categorize` also show the already categorized releases instead of removing them? I'd like to see what I've already categorized releases as, and be able to correct it if I make a mistake.
> 
> Done

I was kind of hoping I could go back after exiting to make a change -- i.e., see all the payloads in the time frame I asked for. Maybe categorize could also take an `-a` option? We can add it later, though, no reason to hold up on that. 

/lgtm

**Comment by dgoodwin** [comment]
Thanks for this Ken, looking forward to using it.

**Pull request merged**